pkgs/development/node-packages/default.nix

```diff
@@ -326,5 +326,14 @@ let
         --prefix NODE_PATH : ${self.prettier}/lib/node_modules
       '';
     };
+
+    wavedrom-cli = super.wavedrom-cli.override {
+      nativeBuildInputs = [ pkgs.pkg-config self.node-pre-gyp ];
+      buildInputs = with pkgs; [
```

Comment:
It would helpful to add a comment explaining that the dependencies are required by https://github.com/Automattic/node-canvas. It took a bit of digging for me to find out.

Reply:
This should be moved into node-canvas that other packages using it benefit from it, too.

Reply:
perhaps we could leave that to a separate PR

Reply:
@SuperSandro2000 IIRC there's no easy way to fix intermediate dependencies with node2nix, so the easiest way to fix them would be in the leaf packages. I might be mistaken though.